hashicorp/waypoint
# ui: 0.7 cleanup (fix dark mode styling, spacing on timeline comp, etc.)

## Included in this PR
- [x] Clean up styling in dark mode (ensure all coloring looks correct)
- [x] Add additional space after the check mark icons of each line in timeline component
- [x] Fix breadcrumbs (seems like most of them except for deployments are off)
- [x] Page headers with icon are missing (build and release details pages)

## How to test
1. Pull down branch `git checkout ui/cleanup-0.7`
2. `yarn start`
3. Click around and check that the things state above look correct

## Screenshots
### Before (Dark Mode)
<img width="1437" alt="Screen Shot 2022-01-11 at 12 50 36 PM" src="https://user-images.githubusercontent.com/60155296/148995507-34f7a5ca-df27-4aa1-ac3a-baf624e7f50c.png">
^ Also note the workspace switcher coloring

<img width="1440" alt="Screen Shot 2022-01-11 at 12 52 11 PM" src="https://user-images.githubusercontent.com/60155296/148995513-43d120f9-260c-4010-8a5e-d3409a05fc34.png">
^ Note breadcrumbs

### After (Dark Mode)
<img width="1440" alt="Screen Shot 2022-01-11 at 12 45 23 PM" src="https://user-images.githubusercontent.com/60155296/148994923-74a0ebd4-7117-4bcf-8d7b-7ca19a1a027d.png">
<img width="1440" alt="Screen Shot 2022-01-11 at 12 46 37 PM" src="https://user-images.githubusercontent.com/60155296/148994926-8215edb2-bbb7-4afb-b303-88e3394bc03c.png">
^ Note breadcrumbs
